### PR TITLE
This is the third milestone for the Project React - Calculator

### PR DESCRIPTION
**Pull request by redacuve**
This PR contains all of the third milestone requirements:

- Use flex display mode for App, Display and ButtonPanel components
- Style the Display component
- Style the ButtonPanel component
- Style the Button component
- Pass “color” and “wide” props to the Button

This milestone is already deployed in Heroku, you can check this here: https://calculator-react-project.herokuapp.com/.

It has two files for the design, the App.css file is where all of the design of this PR is, and the file index.css contains only the style applied to the body and the root div (for centering the calculator).

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1 +1,30 @@
-/* stylelint-disable no-empty-source */
+.app {
+  width: 700px;
```
Comment (Oghenebrume50) [inline src/App.css:4]: should use flex display see

File: src/App.css [hunk 2]
```diff
@@ -1 +1,30 @@
...
+}
+
+.display {
```
Comment (Oghenebrume50) [inline src/App.css:7]: Should use flex display see

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi @redacuve nice job :+1:  this looks really good but I have pointed out some things that can be changed in order for this to pass. Please look into and make changes :slightly_smiling_face: 

**Please make changes and request for another review** :slightly_smiling_face: 

**RAPHAEL NORIODE** :mask:  stay_safe

**Review by shshamim63 — APPROVED** [review]
## :100: Approved :+1: 
___
Hi @redacuve :wave: 
You have covered all the issues raised by the previous reviewer. I formation of your Readme is well organized, neat and clean. The PR contains all the necessary info to help the reviewer. This is way professional. Great job :clap: 
____
#### This :+1: is from me. You can go ahead.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

**Comment by redacuve** [comment]
A big thank you @shshamim63, have a great day :)

**Pull request merged**